app/dashboards/comedian_dashboard.rb

```diff
@@ -69,7 +69,7 @@ class ComedianDashboard < Administrate::BaseDashboard
   # Overwrite this method to customize how comedians are displayed
   # across all pages of the admin dashboard.
   #
-  # def display_resource(comedian)
-  #   "Comedian ##{comedian.id}"
-  # end
+  def display_resource(comedian)
```

Comment:
Doesn't depend on instance state (maybe move it to another class?) (UtilityFunction)